src/Directory.Packages.props

```diff
@@ -15,6 +15,7 @@
     <PackageVersion Include="StyleCop.Analyzers" Version="1.2.0-beta.435" />
     <PackageVersion Include="System.ComponentModel.Annotations" Version="4.5.0" />
     <PackageVersion Include="System.Diagnostics.DiagnosticSource" Version="7.0.0" />
+    <PackageVersion Include="System.Threading.RateLimiting" Version="7.0.0" />
```

Comment:
Let's add this to the dependabot ignores:

https://github.com/App-vNext/Polly/blob/2627c4518692020274c5b4a9e17dff7624fa1f3d/.github/dependabot.yml#L17-L24